pkg/splunk/enterprise/indexercluster.go

```diff
@@ -589,7 +589,7 @@ func getIndexerStatefulSet(client splcommon.ControllerClient, cr *enterprisev1.I
 	// 1. Introduce the new env variables in the function getIndexerExtraEnv
 	// 2. Avoid SPLUNK_INDEXER_URL in getIndexerExtraEnv for idxc CR
 	// 3. Re-introduce the call to getIndexerExtraEnv here.
-	return getSplunkStatefulSet(client, cr, &cr.Spec.CommonSplunkSpec, SplunkIndexer, cr.Spec.Replicas, make([]corev1.EnvVar, 0))
+	return getSplunkStatefulSet(client, cr, &cr.Spec.CommonSplunkSpec, SplunkIndexer, cr.Spec.Replicas, cr.Spec.CommonSplunkSpec.ExtraEnv)
```

Comment:
Is the default value of `cr.Spec.CommonSplunkSpec.ExtraEnv` an slice of type `corev1.EnvVar` and size 0?

Reply:
Yes.